### ISSUE TITLE
Internal server error showing dashboard page

### ISSUE THREAD

**Issue by svenpopping** [WISVCH/payments#55]
### Expected behavior
Showing the dashboard of payments when going to [https://ch.tudelft.nl/payments/dashboard/](https://ch.tudelft.nl/payments/dashboard/)

### Actual behavior
Getting a `500 - Internal server error`

```
2017-09-08 19:43:41.398  WARN 22 --- [nio-8080-exec-8] o.h.e.j.s.SqlExceptionHelper             : SQL Error: 0, SQLState: 42601
2017-09-08 19:43:41.398 ERROR 22 --- [nio-8080-exec-8] o.h.e.j.s.SqlExceptionHelper             : ERROR: syntax error at or near "order"
  Position: 350
2017-09-08 19:43:41.401 ERROR 22 --- [nio-8080-exec-8] o.a.c.c.C.[.[.[.[dispatcherServlet]      : Servlet.service() for servlet [dispatcherServlet] in context with path [/payments] threw exception [Request processing failed; nested exception is org.springframework.dao.InvalidDataAccessResourceUsageException: could not extract ResultSet; SQL [n/a]; nested exception is org.hibernate.exception.SQLGrammarException: could not extract ResultSet] with root cause

org.postgresql.util.PSQLException: ERROR: syntax error at or near "order"
  Position: 350
```

### Steps to reproduce behavior
Just go to [https://ch.tudelft.nl/payments/dashboard/](https://ch.tudelft.nl/payments/dashboard/)

**Comment by skamoen** [comment]
I don't have access anymore 😓

**Comment by skamoen** [comment]
Do you have a copy of all (non-sensitive) environment variables? Syntax errors shouldn't really happen with Hibernate

**Comment by svenpopping** [comment]
```
        - name: SERVER_PORT
          value: "8080"
        - name: SERVER_CONTEXT_PATH
          value: /payments
        - name: JAVA_OPTS
          value: -XX:+UnlockExperimentalVMOptions -XX:+UseCGroupMemoryLimitForHeap
            -XX:MaxRAMFraction=1 -XX:+ExitOnOutOfMemoryError -XshowSettings:vm -Djava.security.egd=file:/dev/./urandom
        - name: SPRING_PROFILES_ACTIVE
          value: prod
        - name: SPRING_MAIL_HOST
          value: ch.tudelft.nl
        - name: SPRING_DATASOURCE_URL
          value: jdbc:postgresql://joost.chnet/payments?ssl=true
        - name: PAYMENTS_WEBHOOKURL
          value: https://ch.tudelft.nl
        - name: PAYMENTS_RETURNURL
          value: https://ch.tudelft.nl
        - name: PAYMENTS_MAIL_SENDER
          value: payments@ch.tudelft.nl
        - name: CONNECT_ADMIN_GROUPS
          value: chbeheer,bestuur,hoothub
        - name: CONNECT_ISSUER_URI
          value: https://connect.ch.tudelft.nl/
        - name: CONNECT_CLIENT_URI
          value: https://ch.tudelft.nl/payments
```

**Comment by skamoen** [comment]
Fixed in b09e96faa51d9ede1fd4618a10dff9ac1a85c999
